agent/pkg/tunnel/tunnel.go

```diff
@@ -48,12 +50,16 @@ func (t *TunnelAgent) Run() {
 				}
 
 				klog.Infof("agent success connected to server %v", relay.Addrs)
+				isSuccess = true
 				break
 			}
 		}
 		// heartbeat time
```

Comment:
What does heartbeat for if you break the loop?

Reply:
Actually I think the heartbeat is useless, and I have delete the code.

Reply:
if this, then need to rethink the whole loop.